packages/log-structured/index.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const { Transform } = require('stream');
+
+const write = (callback, line) => {
+    if (typeof line === 'string' || Buffer.isBuffer(line)) {
+        return callback(null, `${line.toString()}`);
+    }
+
+    return callback(null, `${JSON.stringify(line)}\n`);
+};
+
+const transformRequest = (log) => {
+    const updated = Object.assign({}, log);
+    const { req, res } = updated;
+
+    updated.httpRequest = {
+        protocol: req.protocol,
+        referer: req.headers.referer || '',
+        remoteIp:
+            req.headers['x-forwarded-for'] ||
+            // This is for < v13
+            req.remoteIp ||
+            // This is for > v14
+            '',
+        requestMethod: req.method,
+        requestSize: req.headers['content-length'] || 0,
+        requestUrl: req.url,
+        responseSize: res.size || 0,
+        status: res.statusCode,
+        userAgent: req.headers['user-agent'] || ''
+    };
+
+    return updated;
+};
+
+const parse = (input) => {
+    try {
+        return JSON.parse(input);
+    } catch (e) {
+        return input;
+    }
+};
+
+module.exports = () => {
+    return new Transform({
```

Comment:
Do you need the extra `return` here? Couldn't you just return it directly?

Reply:
Whoops, I forgot to change that back.